lib/repo_tools.pm

```diff
@@ -375,4 +377,35 @@ sub generate_version {
     return $dist . $separator . $version;
 }
 
+
+=head2 validate_repo_enablement
+
+ validate_repo_enablement(%args);
+
+Validates that repo with give name and alias has correct uri and is enabled.
```

Comment:
Could you please fix misspell in 'give' to 'given'.

Reply:
Sure, thanks!